package.json

```diff
@@ -64,5 +65,9 @@
       "type": "MIT",
       "url": "http://github.com/Raynos/fake-ses/raw/master/LICENSE"
     }
-  ]
+  ],
+  "dependencies": {
+    "mkdirp": "^0.5.5",
```

Comment:
nodejs has `fs.mkdir(folder, { recursive: true })` so this dependency is not needed.